modelon/impact/client/experiment_definition/operators.py

```diff
@@ -253,10 +274,44 @@ def __str__(self) -> str:
 
     def to_dict(self, name: str) -> dict[str, Any]:
         return {
-            "kind": "beta",
+            "kind": "normal",
             "name": name,
             "mean": self.mean,
             "variable": self.variance,
             "start": self.start,
             "end": self.end,
         }
+
+    @classmethod
+    def from_dict(cls, data: dict[str, Any]) -> Normal:
+        return Normal(
+            mean=data["mean"],
+            variance=data["variance"],
+            start=data["start"],
+            end=data["end"],
+        )
+
+
+def get_operator_from_dict(
+    data: dict[str, Any]
+) -> Union[Range, Choices, Uniform, Beta, Normal, Enumeration, int, float, bool, str]:
+    kind = data["kind"]
+    if kind == "range":
+        return Range.from_dict(data)
+    elif kind == "choices":
+        return Choices.from_dict(data)
+    elif kind == "uniform":
+        return Uniform.from_dict(data)
+    elif kind == "beta":
+        return Beta.from_dict(data)
+    elif kind == "normal":
+        return Normal.from_dict(data)
+    elif kind == "value":
+        data_type = DataType(data["dataType"])
+        if data_type == DataType.ENUMERATION:
+            return Enumeration(data["value"])
+        elif data_type == DataType.BOOLEAN:
+            return data["value"] == "true"
```

Comment:
This looks strange. I am pretty sure `data["value"]` is just a boolean and not a string? So this elif can be removed.

Reply:
You are right! :)